src/java.xml/share/classes/com/sun/org/apache/xalan/internal/xsltc/trax/TransformerFactoryImpl.java

```diff
@@ -216,7 +215,7 @@ public PIParamWrapper(String media, String title, String charset) {
     /**
      * <p>State of secure processing feature.</p>
      */
-    private boolean _isNotSecureProcessing = true;
+    private boolean _isNotSecureProcessing = false;
```

Comment:
An unfortunate double-negative state for the feature.
(Makes it hard to understand and easy to get confused about).